WalletWasabi/WabiSabi/Models/ConnectionConfirmationResponse.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WalletWasabi.WabiSabi.Crypto;
+
+namespace WalletWasabi.WabiSabi.Models
+{
+	public class ConnectionConfirmationResponse
+	{
+		public ConnectionConfirmationResponse(RegistrationResponseMessage amountCredentials, RegistrationResponseMessage weightCredentials)
```

Comment:
The coordinator has no way to distinguish between a amount credential and a weight credential so, how are we going to prevent a malicious client to use the weight credentials as amount credentials to request more money than what it registered?

We need a cryptographic way to make them impossible to mix.

Reply:
Yeah, I added exceptions, but will make it nicer soonish.

Reply:
Done

Reply:
I realized I misunderstood your comment here, but there's no issue, because the `CredentialIssuerSecretKey` is different for amount and weight credentials.